### PR TITLE
Meet the teams: Insurance pod

### PR DESCRIPTION
**Pull request by DuncanFenning**
### Blog title

Meet the teams: Insurance pod

### Blog description

New meet the team series, an insight to the pods in the web team. This is the first in the series, hopefully to be followed by a blog post from each pod. Any suggestions welcome to changes in format/questions etc

Answers for each question come from the person asked

#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [ ] I have checked for grammatical errors
- [ ] I have checked for spelling errors
- [ ] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

File: _posts/2017-07-14-meet-the-team-insurance.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+layout: post
+title: "Meet the Team: Insurance Pod"
+categories: team
+author: duncan_fenning
+tags: team tech profiles insurance pod
+comments: true
+excerpt: "It's the third edition of our Meet the Team mini series! We thought we would introduce some of our pods, starting with Insurance."
+---
+
+At Holiday Extras, our Web Team works in pods. Each pod is a small team, of differing sizes, each with a strategic goal to work towards. Each pod consists of different roles, including a Pod Lead, an Agile Coach, multiple Software Engineers and Software Testers, as well as any other role required to ensure the pod meets its required strategic objective.
```
Comment (flyingbuddha) [inline _posts/2017-07-14-meet-the-team-insurance.md]: Will this paragraph carry forward into future _meet the teams_ blog posts? I think it summarises what the pods are very well.

File: _posts/2017-07-14-meet-the-team-insurance.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+**Time at HX:** I have been at HX for just over a year.
+
+**What is the biggest challenge you face in your role?**
+
+Being an agile coach means that you need to be really flexible in your approach and be able to pivot and respond to change quickly. Sometimes this can mean changing teams and this requires you to get up to speed with a quick turnaround. Although it is a challenge it is also one on the things I love most about my role.
+
+**What do you most enjoy about the travel insurance business?**
+
+Travel insurance is an incredibly complex product and there is SO much to learn. I enjoy being able to work with lots of different teams disciplines and being able to understand the impact of the tech work that we are delivering.
```
Comment (flyingbuddha) [inline _posts/2017-07-14-meet-the-team-insurance.md]: Should:
`different teams disciplines`

Be:
`different team disciplines`

?

File: _posts/2017-07-14-meet-the-team-insurance.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+**Time at HX:** 6 years
+
+**What is the biggest challenge you face in your role?**
+
+This would probably be the current tech that sits behind the Insurance Product.
+This can make it very difficult to ship solutions quickly and maintain the product.
+
+**What do you most enjoy about the travel insurance business?**
+
+Challenges everyday. There are constantly new things to learn about Insurance as it evolves at a very fast pace. What I enjoy the most, is that it is such a large and complex product, that you will never no everything there is to know, so everyday is filled with something new.
```
Comment (flyingbuddha) [inline _posts/2017-07-14-meet-the-team-insurance.md]: Change:
`that you will never no everything`

To:
`that you will never know everything`

Consider rewriting the paragraph to have less commas, i.e.:
```
I enjoy that Insurance is such a large and complex product, which means you never know everything there is to know. Everyday is filled with something new
```

File: _posts/2017-07-14-meet-the-team-insurance.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+
+Growth is limited by our capability to deliver and market/sell. Not physical stock.
+
+**What do you most enjoy about working in tech?**
+
+Constant change, 20% of what you know becomes irrelevant every year.
+
+**Give us a random fact about yourself**
+
+I would constantly travel. The world is a book and those who do not travel read only one page.
```
Comment (flyingbuddha) [inline _posts/2017-07-14-meet-the-team-insurance.md]: Is the word `would` intentional?
It changes the meaning from _something that is done_ and _something that Hein would like to do_

The second sentence implies that Hein actually does travel.

File: _posts/2017-07-14-meet-the-team-insurance.md [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+
+Problem solving. It's part of my role and its something I find myself doing everyday. Luckily for me, I find myself making the most of my problem solving skills outside of work, so I'm grateful for the challenge!
+
+**What do you most enjoy about the travel insurance business?**
+
+The diversity of the work is what I enjoy most. We get to work with so many different teams, each part of the wider Insurance business. No two days are the same and I find myself meeting lots of new people and learning all the time. It keeps me motivated!
+
+**What do you most enjoy about working in tech?**
+
+For me it has to be the people. Being a bit of a geek myself, I feel at home when working alongside others who enjoy tech as much as I do. We have a load of fun but at the same time, share a can do attitude to getting things done. I never feel demotivated!
```
Comment (flyingbuddha) [inline _posts/2017-07-14-meet-the-team-insurance.md]: Change:
`can do attitude`

To:
`can-do attitude`

File: _posts/2017-07-14-meet-the-team-insurance.md (hunk 1 above)
Comment (DuncanFenning) [inline _posts/2017-07-14-meet-the-team-insurance.md]: Perhaps, although wouldnt want it to get repetitive. I think its useful to have somewhere, maybe other blogs can link to this blog post and say, `for a breakdown of what our pods consist of, check out... `

File: _posts/2017-07-14-meet-the-team-insurance.md (hunk 4 above)
Comment (DuncanFenning) [inline _posts/2017-07-14-meet-the-team-insurance.md]: ah I still don't have a random fact from Hein. I copied this from the meet the sofians blog post (shhhhh). I'll update with Hein's response

**Pull request merged**